pages/common/rclone.md

```diff
@@ -0,0 +1,35 @@
+# rclone
+
+> CLI program to copy/sync/move files and directories to and from many cloud services.
+
+- List contents of a folder on an rclone remote:
+
+`rclone lsf {{remote_name}}:{{path/to/folder}}`
+
+- Copy file or folder from local source to remote destination:
+
+`rclone copy {{/path/to/source_file_or_folder}} {{remote_name}}:{{path/to/destination_folder}}`
+
+- Copy file or folder from remote source to local destination:
+
+`rclone copy {{remote_name}}:{{path/to/source_file_or_folder}} {{/path/to/destination_folder}}`
```

Comment:
Remove preceding `/` from `/path` here and elsewhere.

Reply:
In some cases the preceding `/` is required, and in some it shouldn't be used. All commands here were tested as shown, and are confirmed to work, so I wouldn't recommend removing any slashes.

Reply:
Then in cases where `/` is required, we should move the `/` out of the tokens (`{{}}`).

Reply:
I just did that and was about to commit the change, but when I saw how it actually looks, I stopped. `/` is an integral part of the path, and placing it outside of the double braces makes it appear to be part of the command's syntax, which is not the case. I think that's why we don't see things like `rm` **/**`var/logs/my.log`. 

So are you really sure about this request?

Reply:
Changes made.

Reply:
Umm.. I don't see the `/` outside the token. Am I missing something ?

Reply:
Do you not see [this](https://github.com/tldr-pages/tldr/pull/2610/commits/ed968e43634f405837a8699f56648197417e40ff)? Not a big github user, so I may have neglected to do something besides make the edits.

Reply:
By the way, in light of your comment on not using absolute paths, I decided to eliminate _all_ leading slashes, though I don't think this is a good solution.

`rclone` does _not_ take a leading slash before the first directory in a remote share (so it's `remote:home/user/bin`, even though the same path locally is `/home/user/bin`). The absence of something can't be shown, of course, so the only way to make this clear in an example is to put the leading slashes in all _non_-remote directories, so that their absence in remotes is obviously intentional, and communicates the idea that they shouldn't be used.

This clashes with the `tldr` style guide. 

So the choice is one of stylistic consistency versus accuracy. I personally think the latter is the best choice, but I suspect that insisting on this point is a losing battle, so I won't insist.

It might be worthwhile to consider just how iron-clad the style rules should be, in general.

Reply:
> By the way, in light of your comment on not using absolute paths, I decided to eliminate all leading slashes

According to this [comment](https://github.com/tldr-pages/tldr/pull/2610#discussion_r236039871) of mine, I thought we decide on pulling the `/` out of the token. But it seems you have removed it. I thought you meant those were required in some cases ?

> So the choice is one of stylistic consistency versus accuracy. I personally think the latter is the best choice, 

Sure, you can just add a line in the sub-heading mentioning this. So that it is extra clear.